site/en/blog/autoplay/index.md

```diff
@@ -0,0 +1,290 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Autoplay policy in Chrome
+description: >
+  Learn best practices for good user experiences with the new autoplay policies in Chrome.
+subhead: >
+  Improved user experience, minimized incentives to install ad blockers, and reduced data consumption
+date: 2017-09-13
+updated: 2021-05-25
+authors:
+  - beaufortfrancois
+tags:
+  - autoplay
+  - media
+---
+
+{% Aside %}The Autoplay Policy launched in Chrome 66 for audio and video
+elements and is effectively blocking roughly half of unwanted media autoplays in
+Chrome. For the Web Audio API, the autoplay policy launched in Chrome 71. This
+affects web games, some WebRTC applications, and other web pages using audio
+features. More details can be found in the [Web Audio API](#webaudio) section
+below.{% endAside %}
+
+Chrome's autoplay policies changed in April of 2018 and I'm here to tell
+you why and how this affects video playback with sound. Spoiler
+alert: users are going to love it!
+
+
+
+
+<figure class="w-figure">
+  {% Columns %}
+
+  {% Column %}
+
+  {% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/iEG2wgEqo8Sx40QqfS8E.jpeg", alt="Liam Neeson: I will find you and I will pause you.", width="398", height="374" %}
+
+  {% endColumn %}
+
+  {% Column %}
+
+  {% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/pl6fnOXI5RWtyXXsfYPg.jpeg", alt="Sean Bean: One does not simply autoplay videoas.", width="335", height="335" %}
+
+  {% endColumn %}
+
+  {% endColumns %}
+  <figcaption class="w-figcaption">
+    Internet memes tagged "autoplay" found on <a href="https://imgflip.com/i/ngd6c">Imgflip</a> and <a href="https://imgur.com/a/p1ZjC">Imgur</a>.
+  </figcaption>
+</figure>
+
+## New behaviors
+
+As you may have [noticed], web browsers are moving towards stricter autoplay
+policies in order to improve the user experience, minimize incentives to install
+ad blockers, and reduce data consumption on expensive and/or constrained
+networks. These changes are intended to give greater control of playback to
+users and to benefit publishers with legitimate use cases.
+
+Chrome's autoplay policies are simple:
+
+- Muted autoplay is always allowed.
+- Autoplay with sound is allowed if:
+    - The user has interacted with the domain (click, tap, etc.).
+    - On desktop, the user's [Media Engagement Index](#media-engagement-index)
+      threshold has been crossed, meaning the user has previously played video
+      with sound.
+    - The user has [added the site to their home screen] on mobile or
```

Comment:
This link does not work. Please update with https://web.dev/customize-install/

Reply:
Fixed.